cfg/2.0.0/definitions.yaml

```diff
@@ -7242,8 +7360,8 @@ groups:
       audit: "stat /etc/at.deny"   
       tests:
         test_items:
-        - flag: "stat: cannot stat '/etc/at.deny': No such file or directory"
-          set: true                        
+        - flag: "File: /etc/at.deny"
```

Comment:
Same comment as above

Reply:
apparently it didn't work perfectly before, so I thought about going with the Process of elimination, this is why I switched the set flag into false as well.

Reply:
Same comment as as above - looking for "No such file or directory" might be a safer way to do this

Reply:
Hey you are right and fixed here, https://github.com/aquasecurity/bench-common/pull/58.
I will now update the yaml too